Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -290,7 +290,7 @@ protected Subscription CreateDataSubscription(SubscriptionRequest request)
                                 else
                                 {
                                     var tick = data as Tick;
-                                    if (tick.TickType == TickType.Trade)
+                                    if (tick?.TickType == TickType.Trade && !tick.Suspicious)
```

Comment:
Unrelated, but perhaps providing a great argument for the existing solution, other downstream components such as [fill models](https://github.com/QuantConnect/Lean/pull/2763) would all likely want to duplicate this suspicious check, and also, and custom user models would likely forget to include this check -- so perhaps the high-level suspicious check is best... in fact... have we ever seen someone analyze the suspicious ticks?

Reply:
We made the historical bars have suspicious ticks stripped out but the tick stream is there for experts who want the unfiltered version. The aim was for the live feed to mirror this, removing the bad ticks for bars but keeping them for tick data experts.

Reply:
Just to clarify, this is the current situation before this PR:

Backtesting:
- Tick resolution equity data files contain all ticks, with the last column set to 1 for bad ticks, which corresponds to the `tick.Suspicious` property set to `true`
- Lower resolution equity data files include TradeBars generated only from non-suspicious ticks
- Tick consolidators include all ticks (suspicious or not) ==> BUG, fixed in this PR

Live:
- Subscriptions at Tick resolutions pass all ticks through to the algorithm with no filter
- Subscriptions at lower resolutions include all ticks, which will be used for aggregation into bars ==> BUG, fixed in this PR
- Tick consolidators include all ticks (suspicious or not) ==> BUG, fixed in this PR

@mchandschuh The reason for the filtering in the `LiveTradingDataFeed` was basically what you wrote in the second comment (to avoid duplicating the check in many other places).

As @jaredbroad said, tick subscriptions are intentionally unfiltered, but a fix for tick consolidators was still required (in `TimeSliceFactory`).